src/main/java/com/google/sps/data/MemeDatabase.java

```diff
@@ -0,0 +1,65 @@
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.Query;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.*;
```

Comment:
Prefer to import the specific things you need and not .*

Importing lots of stuff can make your binary size increase which makes builds take longer and deployments be more fickle.

Reply:
+1